console/src/domain/dataset/sdk.ts

```diff
@@ -103,7 +104,10 @@ export class DatasetObject {
             const attr = key.replace(/^_annotation?_/, '')
 
             try {
-                const annos = JSON.parse(value, undefined)
+                const annos =
+                    String(value).startsWith('{') || String(value).startsWith('[')
+                        ? JSON.parse(value, undefined)
+                        : value
```

Comment:
Just try with JSON.parse and catch the exception?
```shell
node
Welcome to Node.js v16.18.0.
Type ".help" for more information.
> JSON.parse('foo')
Uncaught SyntaxError: Unexpected token o in JSON at position 1
> JSON.parse('{"foo": "bar"}')
{ foo: 'bar' }
> JSON.parse('{foo')
Uncaught SyntaxError: Unexpected token f in JSON at position 1
```

Reply:
case that unexpected

![000d57ea-823b-4cf9-9663-39eca3e33c9b](https://user-images.githubusercontent.com/100347187/201583859-6b9a9450-daf5-435b-89c3-3bf436d2271c.jpeg)
![image](https://user-images.githubusercontent.com/100347187/201584649-07e7517a-0233-4b77-a7f2-e1d5482b91b9.png)